vampyr/functions/AnalyticFunction.cpp

```diff
@@ -0,0 +1,30 @@
+#include <array>
+
+#include "pybind11/eigen.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/functional.h"
+#include "pybind11/stl.h"
+
+#include "PyRepresentableFunction.h"
+
+#include "functions/AnalyticFunction.h"
+
+using namespace mrcpp;
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace vampyr {
+
+void analytic_function(py::module &m, py::class_<RepresentableFunction<3>, PyRepresentableFunction<3>> &repfunc) {
+    const auto D = 3;
+
+// The double * wont work properly in vampyr since raw pointers does not exist in python. 
```

Comment:
@bjorgve is this comment still valid? To make it work with `double *`, one should probably use the `py::buffer` (or better `py::array_t<double>`) but I think the best path forward is to move to `std::vector` in MRCPP.

Reply:
I am not familiar with `py::buffer` or `py::array_t<double>`. But previously when this problem has been handled by using `std::array` or `std::vector` on the `MRCPP` side.

Reply:
Indeed, that is the solution to be preferred.